uportal-war/src/main/java/org/jasig/portal/layout/dao/jpa/StylesheetUserPreferencesImpl.java

```diff
@@ -83,6 +84,7 @@
 class StylesheetUserPreferencesImpl implements IStylesheetUserPreferences {
     @Id
     @GeneratedValue(generator = "UP_SS_USER_PREF_GEN")
+    @Index(name = "IDX_SS_USER_PREF_ID")
     @Column(name = "SS_USER_PREF_ID")
     private final long id;
     
```

Comment:
In Aaron's email he mentioned `up_ss_user_pref_lay_attr` table. This column is already indexed on this table (`up_ss_user_pref`) and the other. Was there another column that should be indexed?

Reply:
Nope, sorry I messed up :)